tools/schema-generator.py

```diff
@@ -442,11 +440,14 @@ def handle_attributes(resource_schema, attributes):
         for _, resource in group.get("resources", {}).items():
             resource = resolve_resource(group, resource)            
             resource_name = resource["singular"]
-
+            props = {}
+            props[resource_name+"id"] = {"type": "string", "description": f"ID of the {resource_name} object"}
+            props.update(copy.deepcopy(json_common_attributes))
+            
             if resource.get("hasdocument", True):
-                resource_schema = {
+                 resource_schema = {
```

Comment:
I think this was accidental. We now have this at 5 spaces of indent, then 3 spaces in the next line.

Reply:
I'm confused. I see 4 space indents being used on the lines around this one.

Reply:
They were, yes - but after the change, there's 5 here (and 3 below). `if re` is 5 letters, whereas the indent should just be lined up after `if r`.

Reply:
oh sorry - I thought this was my PR (not Clemens') so I was looking at an untouched version of this file.

Reply:
Fixed